src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/types/ValidateApplicantPartyIdResponse.java

```diff
@@ -12,7 +12,7 @@
  *
  */
 @XmlRootElement
-@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
Do we need to change on dps validation too ?

Reply:
Probably since the ORDS response comes back with empty strings for values which are not returned (empty).